paddle/fluid/operators/fc_op.cc

```diff
@@ -110,6 +119,25 @@ void FCOpMaker::Make() {
   AddInput("W", "(Tensor), The weight fc op with shape (I, O).");
   AddInput("Bias", "(Tensor, optional) Bias vector with shape (1 x O")
       .AsDispensable();
+
+  AddAttr<float>("Scale_in",
+                 "(float, default 1.0f), The quantize scale of input data")
+      .SetDefault(1.0f);
+  AddAttr<std::vector<float>>("Scale_weights",
+                              "(std::vector<float>, default {1.0f}), The "
+                              "quantize scale of weights data")
+      .SetDefault({1.0f});
+  AddAttr<float>("Scale_out",
+                 "(float, default 1.0f), The quantize scale of output data")
+      .SetDefault(1.0f);
+  AddAttr<std::string>(
+      "data_format",
+      "(string, default NCHW) Only used in "
+      "An optional string from: \"NHWC\", \"NCHW\". "
+      "Defaults to \"NHWC\". Specify the data format of the output data, "
+      "the input will be transformed automatically. ")
+      .SetDefault("AnyLayout");
```

Comment:
FC has `data_format` NCHW or NHWC? What exactly is its default value?

Reply:
The attribute was firstly introduced by @dzhwinter in conv op for cuda device and followed by other ops. Normally for CPU device, the default value is described in attribute, namely AnyLayout. The op kernel will ask MKL-DNN to do the reorder to the performant format, e,g., NHWC or NCHW16C etc.

Reply:
Fc op is a general op which don't need attribute `data_format`. `data_format` NCHW or NHWC is used for images, and Fc op could be used in any other models, such as NLP, DNN. These models don't have channel concepts.

Reply:
Thanks, we understood your point. We will align FC FP32 finally: if FC FP32 does not require the attribute, we will refine the code for sure.